### PR TITLE
Import resilience

### PR DESCRIPTION
**Pull request by k9ert**
If you import a folder which either don't have wallets or devices, the import will fail.

### CODE REVIEW

**Comment by stepansnigirev** [comment]
It is already fixed in master in #430 :
https://github.com/cryptoadvance/specter-desktop/blob/master/src/cryptoadvance/specter/controller.py#L354-L355

**Comment by stepansnigirev** [comment]
Sorry, my bad, actually it's not fixed...

File: src/cryptoadvance/specter/controller.py [hunk 1]
```diff
@@ -351,8 +351,14 @@ def general_settings():
...
-            restore_devices = json.loads(request.form.get("restoredevices", "[]"))
-            restore_wallets = json.loads(request.form.get("restorewallets", "[]"))
+            if request.form['restoredevices'] != "":
+                restore_devices = json.loads(request.form['restoredevices'])
+            else:
+                restore_devices = []
+            if request.form['restorewallets'] != "":
+                restore_wallets = json.loads(request.form['restorewallets'])
+            else:
+                restore_wallets = []
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/controller.py:361]: I think this is a bit cleaner if you don't mind:
```suggestion
            restore_devices = []
            restore_wallets = []
            if request.form.get("restoredevices", ""):
                restore_devices = json.loads(request.form.get("restoredevices", "[]"))
            if request.form.get("restorewallets", ""):
                restore_wallets = json.loads(request.form.get("restorewallets", "[]"))
```

**Comment by k9ert** [comment]
Well, don't know why the get(key,alternative) is not working but if this one does it ...

**Comment by stepansnigirev** [comment]
get(key, alternative) is not working because key is actually in the form, but it's an empty string. And json parser can't parse an empty string...

**Review by stepansnigirev — APPROVED** [review]

**Pull request merged**